lib/msal-browser/src/utils/BrowserUtils.ts

```diff
@@ -5,20 +5,37 @@
 import { INetworkModule } from "@azure/msal-common";
 import { FetchClient } from "../network/FetchClient";
 import { XhrClient } from "../network/XhrClient";
+import { BrowserAuthError } from "../error/BrowserAuthError";
 
 /**
  * Utility class for browser specific functions
  */
 export class BrowserUtils {
 
-    // #region Window Navigation
+    // #region Window Navigation and URL management
 
     /**
      * Used to redirect the browser to the STS authorization endpoint
      * @param {string} urlNavigate - URL of the authorization endpoint
```

Comment:
Update this comment block with new parameter.

Reply:
done.